src/pages/clients-corporate-list/forms/ClientTemplateForm.js

```diff
@@ -385,280 +360,204 @@ const ClientTemplateForm = ({
+                          displayField='value'
                           readOnly={editMode && true}
                           values={formik.values}
                           required
                           onChange={(event, newValue) => {
                             if (newValue) {
-                              formik.setFieldValue("industry", newValue?.key);
+                              formik.setFieldValue('industry', newValue?.key)
                             } else {
-                              formik.setFieldValue("industry", "");
+                              formik.setFieldValue('industry', '')
                             }
                           }}
-                          error={
-                            formik.touched.industry &&
-                            Boolean(formik.errors.industry)
-                          }
-                          helperText={
-                            formik.touched.industry && formik.errors.industry
-                          }
+                          error={formik.touched.industry && Boolean(formik.errors.industry)}
+                          helperText={formik.touched.industry && formik.errors.industry}
                         />
                       </Grid>
 
                       <Grid item xs={12}>
                         <ResourceComboBox
-                          endpointId={
-                            CurrencyTradingSettingsRepository.Activity.qry
-                          }
-                          name="activityId"
+                          endpointId={CurrencyTradingSettingsRepository.Activity.qry}
+                          name='activityId'
                           label={_labels.activity}
-                          valueField="recordId"
-                          displayField={["reference", "name"]}
+                          valueField='recordId'
+                          displayField={['reference', 'name']}
                           columnsInDropDown={[
-                            { key: "reference", value: "Reference" },
-                            { key: "name", value: "Name" },
+                            { key: 'reference', value: 'Reference' },
+                            { key: 'name', value: 'Name' }
                           ]}
                           readOnly={editMode && true}
                           values={formik.values}
                           required
                           onChange={(event, newValue) => {
                             if (newValue) {
-                              formik.setFieldValue(
-                                "activityId",
-                                newValue?.recordId
-                              );
+                              formik.setFieldValue('activityId', newValue?.recordId)
                             } else {
-                              formik.setFieldValue("activityId", "");
+                              formik.setFieldValue('activityId', '')
```

Comment:
is this required of the resource combobox does the job ?

Reply:
no